src/sage/groups/libgap_mixin.py

```diff
@@ -968,132 +968,238 @@ def is_isomorphic(self, H):
         """
         return self.gap().IsomorphismGroups(H.gap()) != libgap.fail
 
-
-def minimum_generating_set(G) -> list:
-    """
-    Return a list of the minimum generating set of ``G``.
+def minimum_generating_set(G: GapElement) -> list:
+    r"""
+    Returns the minimum generating
+    set of group G.
 
     INPUT:
 
-    - ``G`` -- a group
+    - ``G`` -- a finite group
 
     OUTPUT:
 
-    A list of GAP objects that generate the group.
+    - a minimum generating set of ``G``
```

Comment:
```suggestion
    A list of GAP objects that generate the group.
```

Reply:
Also, I don't think we should change this as it was suggested by the maintainer of sagemath.